src/notebooks/controllers/vscodeNotebookController.ts

```
@@ -666,6 +667,11 @@ export class VSCodeNotebookController implements Disposable, IVSCodeNotebookCont
                 // Kernel errors would have been handled and displayed
                 return;
             }
+            ex = WrappedError.unwrap(ex);
```

Biome (2.4.15)

[error] Reassigning a catch parameter is confusing.

(lint/suspicious/noCatchAssign)